android-sdk/src/main/java/com/mobilecoin/lib/Amount.java

```diff
@@ -43,6 +43,28 @@
         }
     }
 
+    /**
+     * Constructs native Amount object from the txOutSharedSecret and masked value.
+     *
+     * @param txOutSharedSecret  A {@link RistrettoPrivate} representing the shared secret.
```

Comment:
this has to be RistrettoPublic
```suggestion
     * @param txOutSharedSecret  A {@link RistrettoPublic} representing the shared secret.
```

Reply:
Done.